src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,40 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" (empty {}))
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (+ (get count-map "Queued" 0) (get count-map "Launching" 0))
```

Comment:
Since you're adding, you can get rid of the default and just use `(count-map "Queued")` etc.  If the value doesn't exist, it defaults to `nil`, which will coerce to 0 on adding.  You could then reduce to:

    (apply + (map count-map ["Queued" "Launching"]))